packages/console-types/src/types/providerType.ts

```diff
@@ -56,37 +56,44 @@ const functionalitiesSchema = {
 } as const
 
 const { GIT_PROVIDER, ...OTHER_CAPABILITIES } = CAPABILITIES
-export const providerTypeCapabilitySchema = {
-  oneOf: [
-    {
-      type: 'object',
-      additionalProperties: false,
-      required: ['name'],
-      properties: {
-        name: {
-          type: 'string',
-          enum: Object.values(OTHER_CAPABILITIES),
-        },
-        functionalities: functionalitiesSchema,
-      },
-    }, {
-      type: 'object',
-      additionalProperties: false,
-      required: ['name', 'allowedRepositoryVisibilities'],
-      properties: {
-        name: { const: GIT_PROVIDER },
-        allowedRepositoryVisibilities: {
-          type: 'array',
-          items: {
-            type: 'string',
-            enum: ['private', 'internal', 'public'],
-          },
-          description: 'The visibility levels allowed by this git provider',
-        },
-        functionalities: functionalitiesSchema,
+
+const otherProviderTypeCapabilitySchema = {
+  type: 'object',
+  additionalProperties: false,
+  required: ['name'],
+  properties: {
+    name: {
+      type: 'string',
+      enum: Object.values(OTHER_CAPABILITIES),
+    },
+    functionalities: functionalitiesSchema,
+  },
+} as const
+
+const gitProviderTypeCapabilitySchema = {
+  type: 'object',
+  additionalProperties: false,
+  required: ['name', 'allowedRepositoryVisibilities'],
+  properties: {
+    name: { const: GIT_PROVIDER },
+    allowedRepositoryVisibilities: {
+      type: 'array',
+      items: {
+        type: 'string',
+        enum: ['private', 'internal', 'public'],
       },
+      description: 'The visibility levels allowed by this git provider',
     },
-  ],
+    functionalities: functionalitiesSchema,
+  },
+} as const
+
+const isGitProviderCapability = { type: 'object', properties: { name: { const: GIT_PROVIDER } } } as const
```

Comment:
Why are we implementing isGitProviderCapability? Isn't it the same logic that we have in the provider section?

Reply:
exported here, removed from provider section